config/config_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func TestInitConfig(t *testing.T) {
 	a := assert.New(t)
-	c := InitConfig("proj", "reg", "buck", "table", "prof", "me@foo.example", "0.99.0")
+	c := InitConfig("proj", "reg", "buck", "table", "prof", "me@foo.example", "0.99.0", 0)
```

Comment:
We shouldn't use 0, the first version was version 1 and if there is nothing in the config file (only valid in json) then we assume version 1.